includes/functions.php

```diff
@@ -101,15 +101,23 @@ function load() {
  * @return Container
  */
 function container( Container $container = null ) {
-
 	static $instance;
+	static $initialized = false;
+
+	// Ensure this runs after WordPress initialization (plugins_loaded)
+	if ( ! did_action( 'plugins_loaded' ) ) {
+		add_action( 'plugins_loaded', function() use ( $container ) {
+			container( $container );
+		});
+        return null;
```

Comment:
@mamatharao05 I'm pretty sure that returning null here will result in cases where a module or other code is trying to set a property on the container, but they will get null instead, which will result in an error.

Reply:
"When `add_action('plugins_loaded', function() use ($container) { container($container); }); `  is executed, it registers a callback.

However, this callback will invoke the container() method again, and by that time, the condition `did_action('plugins_loaded')` will return true (since plugins_loaded has already fired). As a result, the `return null` line will never be executed during the second call. Hence will remove this `return null` statement. 

@wpscholar Could you please review this change and confirm if removing the `return null` line is the correct solution?

Reply:
Removing 'return null' did not resolve the deprecation error. I would need to investigate further to identify the root cause of the issue.